src/runtime/TypeManager.cs

```diff
@@ -587,7 +587,7 @@ internal static PyType AllocateTypeObject(string name, PyType metatype)
             // Clr type would not use __slots__,
             // and the PyMemberDef after PyHeapTypeObject will have other uses(e.g. type handle),
             // thus set the ob_size to 0 for avoiding slots iterations.
-            Util.WriteIntPtr(type, TypeOffset.ob_size, IntPtr.Zero);
+            Util.WriteCLong(type, TypeOffset.ob_size, 0);
```

Comment:
Sorry, confused because the other PR also includes this change. I believe it is incorrect. The field seems to have type `Py_ssize_t` in CPython code, which is equivalent to `IntPtr`, not C's `long`.

Reply:
I think you are right, will revert.

Reply:
LGTM if this commit is skipped.